examples/gallery/lines/polygons.py

```diff
@@ -0,0 +1,67 @@
+"""
+Filled polygons
+===============
+
+The :meth:`pygmt.Figure.plot` method allows us to plot geographical data such
+as polygons which are stored in a :class:`geopandas.GeoDataFrame` object. Use
+:func:`geopandas.read_file` to load data from any supported OGR format such as
+a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc. You can also
+use a full URL pointing to your desired data source. Then, pass the
+:class:`geopandas.GeoDataFrame` as an argument to the ``data`` parameter in
+:meth:`pygmt.Figure.plot`, and style the geometry using the ``pen`` parameter.
+To fill the polygons based on a corresponding column you need to set
+``fill="+z"```as well as select the appropriate column using the ``aspatial``
+parameter as shown in the example below.
+"""
+
+import geopandas as gpd
+import numpy as np
+import pygmt
+
+# Read polygon data using geopandas
+gdf = gpd.read_file("https://geodacenter.github.io/data-and-lab//data/airbnb.zip")
+# Automatically get min/max coordinates of polygon set
+bounds = gdf.total_bounds
+# Define an edge in degrees to add in each direction to the min/max coordinates
+edge = 0.02
+
+fig = pygmt.Figure()
+
+fig.coast(
+    region=[bounds[0] - edge, bounds[2] + edge, bounds[1] - edge, bounds[3] + edge],
+    projection="M10c",
+    frame=["af", "+tPopulation of Chicago"],
+    water="lightblue",
+    land="gray70",
+    shorelines="1/1p,gray70",
+)
```

Comment:
What about removing the water/land/shorelines and also the frame:
```suggestion
fig.basemap(
    region=[bounds[0] - edge, bounds[2] + edge, bounds[1] - edge, bounds[3] + edge],
    projection="M10c",
    frame="+tPopulation of Chicago",
)
```

I feel the plot below is more beautiful:
![map](https://github.com/GenericMappingTools/pygmt/assets/3974108/78a83636-7eac-4238-a2fb-3a387411e9db)

Reply:
Agree, the simpler the better!